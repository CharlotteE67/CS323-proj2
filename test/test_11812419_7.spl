int test_7()
{
    int arr[10][10];
    arr[0][0][0] = 1;
}
