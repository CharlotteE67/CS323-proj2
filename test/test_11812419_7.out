Error type 10 at Line 4: applying indexing operator on non-array type variables.